WooCommerce/Classes/ViewRelated/Dashboard/Settings/Beta features/BetaFeaturesViewController.swift

```diff
@@ -0,0 +1,200 @@
+import Storage
+import UIKit
+import Yosemite
+
+/// Contains UI for Beta features that can be turned on and off.
+///
+class BetaFeaturesViewController: UIViewController {
+
+    /// Main TableView
+    ///
+    private lazy var tableView: UITableView = {
+        let tableView = UITableView(frame: .zero, style: .grouped)
+        return tableView
+    }()
+
+    /// Table Sections to be rendered
+    ///
+    private var sections = [Section]()
+
+    private let siteID: Int
+
+    init(siteID: Int) {
+        self.siteID = siteID
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Overridden Methods
+    //
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureNavigationBar()
+        configureMainView()
+        configureSections()
+        configureTableView()
+        registerTableViewCells()
+    }
+}
+
+// MARK: - View Configuration
+//
+private extension BetaFeaturesViewController {
+
+    /// Set the title.
+    ///
+    func configureNavigationBar() {
+        title = NSLocalizedString("Beta Features", comment: "Beta features navigation title")
+    }
+
+    /// Apply Woo styles.
+    ///
+    func configureMainView() {
+        view.backgroundColor = StyleManager.tableViewBackgroundColor
+    }
+
+    /// Configure common table properties.
+    ///
+    func configureTableView() {
+        view.addSubview(tableView)
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        view.pinSubviewToAllEdges(tableView)
```

Comment:
Should this pin to all edge margins? If I recall correctly, and I might be wrong because my autolayout is not as good as it should, in order to get the cell edges right on tablets we should ping to pinSubviewToAllEdgeMargins?

Reply:
Notice the difference with the About screen 😊 
<img src="https://user-images.githubusercontent.com/2722505/64152256-d6f2e100-ce5e-11e9-8f4c-f0869416acc1.png" width="350"/>
<img src="https://user-images.githubusercontent.com/2722505/64152261-d9edd180-ce5e-11e9-9cac-664e9552dfb4.png" width="350"/>

Reply:
ah it's [`tableView.cellLayoutMarginsFollowReadableWidth`](https://developer.apple.com/documentation/uikit/uitableview/1614849-celllayoutmarginsfollowreadablew) which is `false` by default starting in iOS 12 based on [this post](https://useyourloaf.com/blog/readable-width-of-table-view-cells/) - updated in https://github.com/woocommerce/woocommerce-ios/pull/1254/commits/4fc33e87200868c14522b2c83570276b46fb3a7a